requirements.txt

```diff
@@ -0,0 +1,84 @@
+appdirs==1.4.4
+attrs==21.4.0
+beautifulsoup4==4.10.0
+bs4==0.0.1
+cattrs==1.10.0
+certifi==2021.10.8
+charset-normalizer==2.0.9
+click==8.0.3
+colorama==0.4.4
+comtypes==1.1.10
+decorator==5.1.1
+distlib==0.3.4
+docopt==0.6.2
+feedparser==6.0.8
+filelock==3.4.2
+Flask==2.0.2
+forex-python==1.8
+freegames==2.4.0
+future==0.18.2
+gazpacho==1.1
+geocoder==1.38.1
+geographiclib==1.52
+geopy==2.2.0
+idna==3.3
+itsdangerous==2.0.1
+jaraco.context==4.1.1
+Jinja2==3.0.2
+joblib==1.1.0
+Js2Py==0.71
+MarkupSafe==2.0.1
+more-itertools==8.12.0
+MouseInfo==0.1.3
+numpy==1.22.2
+packaging==21.3
+Pillow==9.0.0
+pipwin==0.5.1
+platformdirs==2.4.1
+poetpy==1.1.1
+PyAudio @ file:///C:/Users/User/Downloads/PyAudio-0.2.11-cp39-cp39-win_amd64.whl
```

Comment:
This shouldn't exist as it points to your local folder, and would cause issues on other systems, replace it with PyAudio and its version and check whether it's installing on your system in a different environment using `pip install -r requirements.txt`.

Reply:
I have tried to make the desired changes . Could you please review it now?